Makefile_cc65

```diff
@@ -32,21 +38,20 @@ LIB_OBJECTS = \
 	src/tests.o \
 	src/time.o
 
-.PHONY: all clean cleanall
+.PHONY: all clean cleanall test doc
 
-all: libmega65.a
+libmega65.a: $(LIB_OBJECTS)
+	$(AR65) a $@ $^
+
+all: libmega65.a $(TESTS_PRGS)
```

Comment:
Same as with the other PR: all should be the first target in my mindset, as it is then used if make is invoked without a target.

Reply:
Thanks - we should merge this after #39.